test/e2e/checkpoint_test.go

```diff
@@ -933,18 +935,24 @@ var _ = Describe("Podman checkpoint", func() {
 	})
 
 	It("podman checkpoint and restore container with different port mappings", func() {
-		localRunString := getRunString([]string{"-p", "1234:6379", "--rm", redis})
+		port, err := utils.GetRandomPort()
+		Expect(err).ShouldNot(HaveOccurred())
+		orginalPort := port + GinkgoParallelProcess()
```

Comment:
I'm confused, you get a random port (that is "safe") and then add something to it?